cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php

```diff
@@ -343,43 +343,49 @@ public function print_video_scripts() {
 					$config['fluid'] = true;
 				}
 
-				$code[] = sprintf(
-					'var video%1$s = cld.videoPlayer( "cloudinary-video-%1$s", %2$s );' . "\n",
-					$instance,
-					wp_json_encode( $config )
-				);
+				$cld_videos[ $instance ] = $config;
 
-				// Apply transformations via URL
-				// The "320" is hardcoded here as that the rough 
-				// estimate of pixels that occupy the player controls.
-				if ( isset( $this->config['video_freeform'] ) ) {
-					$onload_code[] = sprintf(
-						'
-	var videoContainer%1$s = document.getElementById( video%1$s.videojs.id_ );
-	var videoElement%1$s = videoContainer%1$s.getElementsByTagName( "video" );
-
-	if ( videoElement%1$s.length === 1 ) {
-		videoElement%1$s = videoElement%1$s[0];
-		videoElement%1$s.src = videoElement%1$s.src.replace( "upload/", "upload/%2$s/" );
-
-		if ( videoElement%1$s.width < 320 ) {
-			video%1$s.controls(false);
-		}
-	}
-', 
-						esc_attr( $instance ), 
-						esc_attr( $this->config['video_freeform'] )
-					);
-				}
 			}
-			
-			// If code was populated, output.
-			if ( ! empty( $code ) ) {
-				wp_add_inline_script( 
-					'cld-player', 
-					implode( $code ) . $this->window_onload_wrapper( implode( $onload_code ) ) 
-				);
+
+			if ( empty( $cld_videos ) ) {
+				return;
 			}
+
+			ob_start();
+			?>
+var cldVideos = <?php echo wp_json_encode( $cld_videos ); ?>;
```

Comment:
Nitpick: could this be indented at the same level as the `?>` above? Maybe there is a reason, it just looks a little strange to me.